Mathlib/NumberTheory/NumberField/Discriminant.lean

```diff
@@ -41,6 +43,50 @@ theorem discr_eq_discr {ι : Type*} [Fintype ι] [DecidableEq ι] (b : Basis ι
   let b₀ := Basis.reindex (RingOfIntegers.basis K) (Basis.indexEquiv (RingOfIntegers.basis K) b)
   rw [Algebra.discr_eq_discr (𝓞 K) b b₀, Basis.coe_reindex, Algebra.discr_reindex]
 
+open MeasureTheory MeasureTheory.Measure Zspan NumberField.mixedEmbedding
+  NumberField.InfinitePlace ENNReal NNReal
+
+theorem _root_.NumberField.mixedEmbedding.volume_fundamentalDomain_latticeBasis :
+    volume (fundamentalDomain (latticeBasis K)) =
+      (2 : ℝ≥0)⁻¹ ^ Fintype.card { w : InfinitePlace K // IsComplex w } *
+        Real.toNNReal (Real.sqrt |discr K|) := by
```

Comment:
Does `NNReal.sqrt` exist? If so, you can use `|discr K|\_+` (subscript +) to take the norm as nnreal, and then you don't need the `Real.toNNReal` later on.

Reply:
I changed the statement and the proof. It does look cleaner that way.